azkaban-common/src/main/java/azkaban/imagemgmt/dto/HPFlowDTO.java

```diff
@@ -0,0 +1,43 @@
+package azkaban.imagemgmt.dto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.validation.constraints.NotBlank;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+
+/**
+ * This is Add HP Flow class for updating high priority flow metadata.
+ */
+public class HPFlowDTO extends BaseDTO {
+  // CSV of flow ids.
+  @JsonProperty("flowIds")
+  @NotBlank(message = "flowList cannot be empty.")
+  private String flowIds;
```

Comment:
Why can't we keep list of flowids instead of comma separate string?

Reply:
We get the CSV as input and eventually convert it to list when needed.